intranet/templates/page_with_header.html

```diff
@@ -97,6 +97,7 @@ <h1>Intranet</h1>
                         <li><a href="{% url 'preferences' %}"><i class="fas fa-cog"></i> Preferences</a></li>
                         <li><a href="{% url 'itemreg' %}"><i class="fas fa-edit"></i> Item Registration</a></li>
                         <li><a href="{% url 'lostfound' %}"><i class="far fa-compass"></i> Lost and Found</a></li>
+                        <li><a href="{%  url 'oauth2_provider:list' %}"><i class="fas fa-key"></i> OAuth</a></li>
```

Comment:
```suggestion
                        <li><a href="{% url 'oauth2_provider:list' %}"><i class="fas fa-key"></i> OAuth</a></li>
```
I know I'm _really_ nitpicking here, but it's nice to have consistent formatting.